PySDM/physics/diffusion_kinetics/lowe_et_al_2019.py

```diff
@@ -2,25 +2,21 @@
 eq. 13-14 in Pruppacher & Klett 2005 for Delta v = 0
 no corrections for thermal conductivity
 """
-import numpy as np
+from PySDM.physics.diffusion_kinetics.pruppacher_and_klett_2005 import PruppacherKlett
 
 
-class LoweEtAl2019:
+class LoweEtAl2019(PruppacherKlett):
     def __init__(self, _):
         pass
 
-    @staticmethod
-    def lambdaD(const, D, T):
-        return D / np.sqrt(2 * const.Rv * T)
-
     @staticmethod
     def lambdaK(const, T, p):  # pylint: disable=unused-argument
         return -1
 
-    @staticmethod
-    def D(const, D, r, lmbd):
-        return D / (1 + 2 * np.sqrt(const.PI) * lmbd / r / const.MAC)
-
     @staticmethod
     def K(const, K, r, lmbd):  # pylint: disable=unused-argument
         return K
+
+    @staticmethod
+    def D(const, D, r, lmbd):
+        return super().D(const, D, r, lmbd, 0.0)
```

Comment:
this will almost surely not work with GPU :( (actually, I doubt it will work with Numba)
I guess we need to set the constant to zero in example code, and we can assert in the LoweEtAl2019 ctor that the constant is actually zero (so if the setting is forgotten, the user would be warned)

Reply:
Would it work to add it to `const`?

```suggestion
        return super().D(const, D, r, lmbd, const.l19_dv)
```

Reply:
Sorry, for not being clear. I expect the `super()` not to work within our backends.
I'd thus suggest inheriting without alteration, and then setting the constant value to zero in Formulae ctor, ie..:
```
formulae = Formulae(constants={'l19_dv': 0})
```
and then in the ctor here do:
```
def __init__(self, const):
    assert const.l19_dv == 0
```

Reply:
Got it!